bots/csr/src/test/java/org/openjdk/skara/bots/csr/CSRBotTests.java

```diff
@@ -162,13 +162,19 @@ void keepLabelForNotApprovedCSR(TestInfo testInfo) throws IOException {
             localRepo.push(editHash, repo.url(), "edit", true);
             var pr = credentials.createPullRequest(repo, "master", "edit", issue.id() + ": This is an issue");
 
+            // Run bot
+            TestBotRunner.runPeriodicItems(bot);
+
+            // The bot should have kept the CSR label
+            assertTrue(pr.labelNames().contains("csr"));
+
             // Add CSR label
             pr.addLabel("csr");
```

Comment:
Adding the label here has no effect.

Reply:
yes, because the bot has already added csr label automatically. I removed this step.